### DISCUSSION TITLE
Dev container in docker for windows?

### DISCUSSION THREAD

**Discussion by luebbe** [WeblateOrg/weblate#5844] [Q&A]
In order to test my work on https://github.com/WeblateOrg/weblate/issues/5813, I'm trying to get Weblate dev-docker up and running in docker for windows.
I didn't get very far, because the steps from rundev.sh can't be reproduced 1:1 on windows. Is this possible at all? And if yes, which parameters would I have to pass in for user_id and group_id?
If this is not possible, I'll try at home later on my raspberry :)

**Comment by nijel** [comment]
I have no clue how this behaves on Windows, but you can first try without that and see if something breaks ;-).

**Reply by luebbe** [reply]
I guess I'll have to fire some shots in the dark then. :)
Building weblate docker on windows fails due to the uid/gid problem. Building weblate docker on a raspberry 3B fails due to: "no matching manifest for linux/arm/v7 in the manifest list entries"

**Reply by nijel** [reply]
See https://github.com/WeblateOrg/docker/issues/968 for the arm/v7 issues.

**Comment by luebbe** [comment]
Could I activate the Github actions for the tests on my fork? Would that be possible? They are turned off by default for some reason. Or would I somehow mess with the originial repo's stats, because they have to be reconfigured?

**Reply by nijel** [reply]
I think actions should be IMHO automatically active on forks. At least I never did anything to enable them.

**Reply by luebbe** [reply]
They are disabled by GH and GH warns me to "understand what I am doing" before enabling them.
I'll enable them. Let's see what happens.

**Comment by kb21032001** [comment]
I know this is an old discussion, but I ended up here searching for help so just thought that there might be others like me. I guess for anyone searching here to try and run it on Windows, I would suggest trying and using WSL2 as it is much more straight forward compared to trying to get it to build and run on Windows without it.
